src/main/java/module/tsforecast/TSPanel.java

```diff
@@ -82,7 +82,7 @@
   public static final Color lightYellow = new Color(255, 255, 200);
 
   private final Icon m_newImage   = ThemeManager.getIcon(HOIconName.MATCHICONS[2]);
-  private final Icon m_startImage = ThemeManager.getIcon(HOIconName.MANUELLSMILIES[7]);
+  private final Icon m_startImage = ThemeManager.getIcon(HOIconName.SMILEYS[7]);
```

Comment:
There are only 6 elements in the array, does this not cause a problem?

Reply:
good catch !